javascript/javascript1/week2/flightBooking.js

```diff
@@ -0,0 +1,30 @@
+//First Part
+
+function getFullname1(firstname, surname)
+{
+const fullName = (firstname + " " + surname);
+return fullName;
+}
+console.log (getFullname1("Adnan" , "Aziz"));
+const fullName1 = getFullname1("Abdul", "Hadi");
+const fullName2 = getFullname1("Abdul" , "Manan");
+console.log(fullName1);
+console.log(fullName2);
+
+//Second Part
+
+function getFullname(firstname, surname, useFormalName) {
+    if (firstname && surname == "")
```

Comment:
I think what you wanted to test is something similar to this.
```suggestion
    if ((firstname =="" )&& (surname == ""))
```

Reply:
don't you think it gives the same result i.e short form of writing the condition?